src/Clock.tsx

```diff
@@ -0,0 +1,69 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  today: Date;
+};
+
+function fixTime(date: Date) {
+  return date.toUTCString().slice(-12, -4);
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(fixTime(this.state.today));
```

Comment:
Don't use state here, it can be outdated
Save `new Date` result to constant and use it in both setState and console.info